### ISSUE TITLE
[INFANT] mri_label_fusion error

### ISSUE THREAD

**Issue by mgxd** [freesurfer/freesurfer#795]
### Platform details:

- FreeSurfer version: 7.1.1 (built off 8b40551f096294cc6603ce928317b8df70bce23e)
- Operating system name and version: Ubuntu Xenial

### Summary
Running into a bug while testing out `infant_recon_all`.

```
LABELFUSION CMD mri_label_fusion -i /input/sub-CC00364XX10/mprage.mask.nii.gz -s /output/work/manseg-Template15-2-sub-CC00364XX10.NIFTYREG.nii.gz /output/work/manseg-Template9-2-sub-CC00364XX10.NIFTYREG.nii.gz /output/work/manseg-Template17-2-sub-CC00364XX10.NIFTYREG.nii.gz /output/work/manseg-Template8-2-sub-CC00364XX10.NIFTYREG.nii.gz -o /output/work/sub-CC00364XX10-BETA0.3-RHO.70.labels.mrf.nii.gz --smooth --rho .70 --beta 0.3 --bf-order 4 --max-lab 3 --unary-weight 5 --verbose -e 2 41 3 42 -e 7 46 8 47 -e 12 51 13 52

Traceback (most recent call last):
  File "/usr/local/freesurfer/python/scripts/mri_label_fusion", line 214, in <module>
    counts = np.sum([label[0] in vol and label[1] in vol for vol in moving_vols])
  File "/usr/local/freesurfer/python/scripts/mri_label_fusion", line 214, in <listcomp>
    counts = np.sum([label[0] in vol and label[1] in vol for vol in moving_vols])
TypeError: 'int' object is not subscriptable
set: No match.
```

It seems the list comprehension here
https://github.com/freesurfer/freesurfer/blob/987f92022c4b9b5737179b45df805ddf4bfcb84a/infant/mri_label_fusion#L214
is trying to index an integer (`label`).

**Comment by mgxd** [comment]
Tangentially, while looking through the `mri_label_fusion` command, the `-e, --exclude` flag is specified 3 times and is being overwritten.

```
mri_label_fusion ... -e 2 41 3 42 -e 7 46 8 47 -e 12 51 13 52
```

Only `12 51 13 52` will be excluded here.

cc-ing @lillazollei

**Comment by ahoopes** [comment]
@mgxd thanks so much for finding this, especially since we're about to cut a new infant release! This is now fixed by 0c540ba.

**Comment by mgxd** [comment]
great! would love a ping whenever you end up cutting the release 😄

**Comment by ahoopes** [comment]
@mgxd there is a new infant version available that's built off of freesurfer 7.1.
